sync/src/chain.rs

```diff
@@ -844,6 +865,14 @@ impl ChainSync {
 		}
 	}
 
+	/// Generic packet sender
+	fn send_packet(&mut self, sync: &mut SyncIo, peer_id: PeerId, packet_id: PacketId, packet: Bytes) {
```

Comment:
This is used to send out blocks/hashes/transactions. Does not change peer `asking` status